lib/sorcery/controller.rb

```diff
@@ -96,7 +96,16 @@ def current_user=(user)
 
       # used when a user tries to access a page while logged out, is asked to login,
       # and we want to return him back to the page he originally wanted.
-      def redirect_back_or_to(url, flash_hash = {})
+      def redirect_back_or_to(...)
+        if Config.use_redirect_back_or_to_by_rails
+          super
+        else
+          warn('[WARNING] `redirect_back_or_to` overrides the method of the same name defined in Rails 7. If you want to avoid overriding, you can set `config.use_redirect_back_or_to_by_rails = true` and use `redirect_to_before_login_path`.')
```

Comment:
Since it is desirable for this setting to default to true in the future, I would like to add that "in a future version, `config.use_redirect_back_or_to_by_rails = true` will become the default."

Reply:
I have changed the message in  https://github.com/Sorcery/sorcery/pull/373/commits/8ebe075834d2954b3a557cf74142babb7869f5e9